net.christianbeier.Gromit-MPX.yml

```diff
@@ -1,10 +1,10 @@
 id: net.christianbeier.Gromit-MPX
 runtime: org.gnome.Platform # using GNOME for the time being as we need dconf for setting hotkeys under Wayland
-runtime-version: '3.38'
+runtime-version: '42'
 sdk: org.gnome.Sdk
 command: gromit-mpx
 finish-args:
-  - --socket=x11
+  - --socket=fallback-x11
```

Comment:
why socket-fallback? (It works better with X11 then with Wayland at the moment...)

Reply:
fallback-x11 does not prevent launching it on X11. If the app is launched in a X window system it'll use it, but if the app is launched under Wayland, we don't need to expose the X11 socket entirely with fallback-x11, ie. X11 will be used provided that the app is launched under X otherwise wayland will be used. See https://github.com/flathub/flathub/issues/1452

You can test if this works by doing `flatpak run --nosocket=wayland net.christianbeier.Gromit-MPX` on the test build below to see if it runs under X11. This will block the wayland socket, so the app will fallback to X if it is under a X window or Xwayland if it is under a wayland session.

Please let me know if this works. I checked that it launches but I can always revert it if something doesn't work.

Reply:
What's the benefit of the fallback thingy? Can't we simply leave this as is ;-) ?

Reply:
It does in fact not support native Wayland, we always use XWayland.

Reply:
Works with this finish arg under XWayland as well.